src/status_im/contexts/preview/quo/main.cljs

```diff
@@ -218,7 +219,9 @@
                        {:name      :account-avatar
                         :component account-avatar/view}]
    :banner            [{:name      :banner
-                        :component banner/view}]
+                        :component banner/view}
```

Comment:
hmm is this a quo component? 
We have another namespace for previewing non Quo components.
src/status_im/contexts/preview/status_im/main.cljs